packages/flutter/lib/src/cupertino/search_field.dart

```diff
@@ -337,12 +343,28 @@ class _CupertinoSearchTextFieldState extends State<CupertinoSearchTextField>
   TextEditingController get _effectiveController =>
       widget.controller ?? _controller!.value;
 
+  ScrollPosition? _ancestorScrollPosition;
+  double _fadeExtent = 0.0;
+  double _maxHeight = 0.0;
+
   @override
   void initState() {
     super.initState();
     if (widget.controller == null) {
       _createLocalController();
     }
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
We chatted about removing this post frame callback so we do not have any frame delays or edge cases that would look off. 👍